opencti-platform/opencti-graphql/src/database/session.js

```diff
@@ -64,6 +65,21 @@ export const findSessions = () => {
   });
 };
 
+// return the list of users ids that have a session activ in the last maxInactivityDuration ms
+export const usersWithActiveSession = (maxInactivityDurationInMs = 60000) => {
```

Comment:
can we use minutes in param instead of ms ?

Reply:
yes, done